From a129aeb333d83a4466ac2c996b84a113405acb4d Mon Sep 17 00:00:00 2001
From: philwo <philwo@google.com>
Date: Mon, 14 Jun 2021 09:10:07 -0700
Subject: [PATCH] Fix #13575: FTBFS with GCC 11 on ARMv8

This supposedly fixes the compiler error reported by the user:

`error: 'numeric_limits' is not a member of 'std'`

Thanks to samcom12 for the report and suggested fix.

RELNOTES: None.
PiperOrigin-RevId: 379291338
---
 third_party/ijar/zlib_client.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/third_party/ijar/zlib_client.h b/third_party/ijar/zlib_client.h
index c4b051e0100c..0a917ff0a99a 100644
--- a/third_party/ijar/zlib_client.h
+++ b/third_party/ijar/zlib_client.h
@@ -16,7 +16,9 @@
 #define THIRD_PARTY_IJAR_ZLIB_CLIENT_H_
 
 #include <limits.h>
+
 #include <limits>
+#include <stdexcept>
 
 #include "third_party/ijar/common.h"
 
